AlphaWallet/UI/SelectCurrencyButton.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  SelectCurrencyButton.swift
+//  AlphaWallet
+//
+//  Created by Vladyslav Shepitko on 15.05.2020.
+//
+
+import UIKit
+
+class SelectCurrencyButton: UIControl {
+
+    private let textLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.setContentHuggingPriority(.required, for: .horizontal)
+        label.setContentCompressionResistancePriority(.required, for: .horizontal)
+        label.font = DataEntry.Font.amountTextField
+        
+        return label
+    }()
+    
+    private let expandImageView: UIImageView = {
```

Comment:
Can you hide this when toggling between fiat/token is not available?

There's a line like this in SendViewController:

```
amountTextField.alternativeAmountLabel.isHidden = true
```

Maybe both of them can be extracted into a new `isAlternativeAmountEnabled` property in `AmountTextField`?

Reply:
> git fetch origin master:master #gets the updated `master` and updates it locally

when i do this there error shows. 
Refusing to fetch into current branch refs/heads/master of non-bare repository

Reply:
You can do it in a few steps:

git checkout master
git pull origin master #safe if you haven’t commit to master manually

Reply:
Remember that origin is probably “upstream” for you

Reply:
> You can do it in a few steps:
> 
> git checkout master
> git pull origin master #safe if you haven’t commit to master manually

did it.
it tells next
 * branch              master     -> FETCH_HEAD
Already up to date.

Reply:
seems like i can push my changes, right?

Reply:
Sure.